fs/config.go

```diff
@@ -179,6 +180,11 @@ func NewConfig() *ConfigInfo {
 		if arg == "--log-level=DEBUG" || (arg == "--log-level" && len(os.Args) > argIndex+1 && os.Args[argIndex+1] == "DEBUG") {
 			c.LogLevel = LogLevelDebug
 		}
+		if strings.HasPrefix(arg, "--verbose=") {
```

Comment:
@ncw 
Thank you very much for review.

Do you mind if I put mountlib documentation in a separate commit?

Also I'd like to note that this early `--verbose=x` support in fs/config.go isn't directly connected with mount helper cli translator? I could split it in yet another commit. Do you mind?

Thank you

Reply:
@ncw 
Thank you very much for review.

Do you mind if I put mountlib documentation in a separate commit?

Also I'd like to note that this early `--verbose=x` parser in `fs/config.go` isn't directly connected with mount helper cli translator and may be split it in yet another commit. Do you agree? Or better keep it squashed?

Thank you